models/DataObject/ClassDefinition/Layout/Fieldset.php

```diff
@@ -55,4 +62,26 @@ public function getLabelWidth()
     {
         return $this->labelWidth;
     }
+
+    /**
+     * @param string $labelAlign
+     *
+     * @return $this
+     */
+    public function setLabelAlign($labelAlign): self
```

Comment:
@blankse I'd prefer to move this code into a trait, so avoid code duplication, see also: 
https://github.com/pimcore/pimcore/blob/46b3bff3c519e8eb3254f8da016edb1ef9482755/models/DataObject/ClassDefinition/Layout/Traits/IconTrait.php#L19